web-ui/src/main/resources/catalog/components/thesaurus/ThesaurusService.js

```diff
@@ -202,6 +202,25 @@
             }
           });
 
+          listOfKeywords.sort(function(k1,k2) {
+            var s1;
+            var s2;
+            if (orderById == "true") {
+              s1 = k1.props.uri;
+              s2 = k2.props.uri;
+            } else {
+              s1 = k1.label.toLowerCase();
+              s2 = k2.label.toLowerCase();
+            }
+            if (s1 < s2) {
+              return -1;
+            }
+            if (s1 > s2) {
+              return 1;
+            }
+            return 0;
```

Comment:
To also sort terms with accent 

![image](https://github.com/user-attachments/assets/8c72c8c6-6af1-4ee3-b665-2cd6729ae914)

Maybe 

```suggestion
            return s1.localeCompare(s2);
```
is better?


![image](https://github.com/user-attachments/assets/837b27f0-b8d4-4c45-af03-4fcdf1d0f346)

Reply:
oh yes, much cleaner